src/main/scala/strawman/collection/mutable/GrowableBuilder.scala

```diff
@@ -0,0 +1,23 @@
+package strawman
+package collection.mutable
+
+import scala.Unit
+
+/** The canonical builder for collections that are growable, i.e. that support an
+  * efficient `+=` method which adds an element to the collection.
+  *
+  * GrowableBuilders can produce only a single instance of the collection they are growing.
+  *
+  * @author Paul Phillips
+  * @version 2.8
+  * @since 2.8
+  *
+  * @define Coll `GrowingBuilder`
+  * @define coll growing builder
+  */
+class GrowableBuilder[Elem, To <: Growable[Elem]](empty: To) extends Builder[Elem, To] {
+  protected var elems: To = empty
+  def clear(): Unit = elems.clear()
```

Comment:
I'm not sure about the logic here.  If `clear()` only does `elems.clear()`, why is `elems` a var?